dev/how-to/deploy/orchestrated/tidb-in-kubernetes/aws-eks.md

```diff
@@ -113,15 +114,14 @@ Apply complete! Resources: 67 added，0 changed，0 destroyed.
 Outputs:
 
 bastion_ip = [
-    52.14.50.145
+  "34.219.204.217",
 ]
-eks_endpoint = https://E10A1D0368FFD6E1E32E11573E5CE619.sk1.us-east-2.eks.amazonaws.com
+default-cluster_monitor-dns = a82db513ba84511e9af170283460e413-1838961480.us-west-2.elb.amazonaws.com
```

Comment:
Suggest to print ports info.

Reply:
It's an enhancement which has not yet been made